include/deal.II/grid/tria_iterator.h

```diff
@@ -165,10 +165,10 @@ class TriaActiveIterator;
  *
  * The accessors provided by the library consists of two groups, determined by
  * whether they access the data of Triangulation objects or
- * DoFHandler/hp::DoFHandler objects. They are derived from TriaAccessor and
- * DoFAccessor, respectively. Each group also has specialized accessors for
- * cells (as opposed to faces and lines) that offer more functionality such as
- * accessing neighbors.
+ * DoFHandler&thinsp;-&thinsp;hp::DoFHandler objects. They are derived from
```

Comment:
That looks uncommon. Why do you want to use `-` instead of `/`?

Reply:
I thought the opposite: hyphens are for linking words together in prose. I don't think I have seen `/`s used to do that.

Reply:
I should get a style manual and read it. Wikipedia says that `/` can be used as a shorthand for 'and' so lets stick with `/` here; you're right.